### PR TITLE
Upgrade to latest rom-4

### PR DESCRIPTION
**Pull request by GustavoCaso**

### CODE REVIEW

**Comment by GustavoCaso** [comment]
Removing the `Transproc` part makes the fail by not returning the keys as symbols, but do we need actually always return the keys as symbols?

File: lib/rom/yaml/schema.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require 'rom/types'
+require 'rom/schema'
+
+module ROM
+  module YAML
+    # Elasticsearch relation schema
```
Comment (cflipse) [inline lib/rom/yaml/schema.rb]: Not elasticsearch

**Comment by cflipse** [comment]
I think we do -- it's a matter of consistency between relations, and having to know that _this_ relation is of _this_ type, and therefore has _these_ keys ... that's way too much cognitive overhead, that has to go into _way_ too many locations -- consider the arguments to joining relation data in the `combine` commands ...

**Comment by solnic** [comment]
I restored symbolizing keys. It's because using query API would require using string keys, which would be really weird. In the future we should improve memory adapter (which this one is based on), so that it uses attribute API in query methods.

File: .rspec [hunk 2]
```diff
@@ -1,2 +1,3 @@
 --color
 --order random
+--require ./spec/spec_helper.rb
```
Comment (wafcio) [inline .rspec:3]: in this case it will be good to remove this line from spec files

**Pull request merged**